src/cpp/wrapper.cc

```diff
@@ -110,16 +110,20 @@ NAN_METHOD(GetMe) {
 
 NAN_METHOD(GetContact) {
 #if defined(V8_MAJOR_VERSION) && V8_MAJOR_VERSION >= 7
-  int index = info[0]->Uint32Value(Nan::GetCurrentContext()).ToChecked();
+  unsigned int index = info[0]->Uint32Value(Nan::GetCurrentContext()).ToChecked();
 #else
-  int index = info[0]->Uint32Value();
+  unsigned int index = info[0]->Uint32Value();
 #endif
 
   AddressBook ab;
   Isolate* isolate = Isolate::GetCurrent();
+  Person* person = ab.getContact(index);
 
   Local<Object> contact = Object::New(isolate);
-  fillPersonObject(isolate, contact, ab.getContact(index));
+
+  if (person != NULL) {
+    fillPersonObject(isolate, contact, person);
```

Comment:
If person is NULL, you will now get a JS object with no members for name etc. Depending on how you want to use it maybe you prefer to make an object with those members set to null/empty string

Reply:
I just tried it on Linux with `getContact(0)` and it returns
```js
{ firstName: '', lastName: '', emails: [], numbers: [] }
```

Reply:
That's good then :)